### PR TITLE
fix: styling changes to scheduled deliveries

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes: #4581 

### Description:
before
Screenshot 2023-02-27 at 13 07 10

after

https://user-images.githubusercontent.com/67699259/221644198-578104ba-6201-4da5-bf79-cb2437949b31.mov

### CODE REVIEW

**Comment by TuringLovesDeathMetal** [comment]
Yay! looks much nicer :) 

@PriPatel what do you think of the `format` included in the `select destination` part? I think that you took a look at this when the format wasn't included yet (that was the only thing that was slightly weird looking to me here)

**Comment by PriPatel** [comment]
yes, i agree :)
`Format` looks like its at the same hierarchy as the 3 main steps.

I would make it look like this: 
Screenshot 2023-02-27 at 12 19 51


- [ ] `on` > `format`
- [ ] `Monday` > `Image`
- [ ] Dropdown style should match as well

**Comment by TuringLovesDeathMetal** [comment]
@PriPatel ya, agreed. It's just looking a bit like there's lots of steps right now, but we're also saying there's 3 steps 😅

**Review by TuringLovesDeathMetal — APPROVED** [review]
Nice! looks great!

File: packages/frontend/src/components/SchedulerModals/SchedulerModalBase/SchedulerModalBase.styles.ts [hunk 1]
```diff
@@ -42,15 +42,25 @@ export const SchedulerDetailsContainer = styled.div`
...
 
-export const SettingsWrapper = styled.div`
-    margin-top: 50px;
+export const ModalTitle = styled.p`
+    margin: 3px 0px;
 `;
 
-export const InputWrapper = styled.div`
+export const StyledSelect = styled(HTMLSelect)`
     margin-bottom: 10px;
```
Comment (IrakliJani) [inline packages/frontend/src/components/SchedulerModals/SchedulerModalBase/SchedulerModalBase.styles.ts:55]: if you add Html select inside FormGroup then you don't need this

File: packages/frontend/src/components/SchedulerModals/SchedulerModalBase/SchedulerForm.tsx [hunk 2]
```diff
@@ -196,177 +199,220 @@ const SchedulerForm: FC<
...
                 />
             </FormGroup>
 
-            {format === 'csv' && (
-                <SchedulerOptions disabled={disabled} methods={methods} />
-            )}
+            <InputGroupWrapper>
+                <Title>3. Select format and destination(s)</Title>
+                <InlinedInputs>
+                    <InlinedLabel>Format</InlinedLabel>
```
Comment (IrakliJani) [inline packages/frontend/src/components/SchedulerModals/SchedulerModalBase/SchedulerForm.tsx:234]: FormGroup provides label and once clicked focuses on the element underneath

**Review by IrakliJani — COMMENTED** [review]
code looks good, just adding FormGroup-s might improve UX and code. if not, I'm ok to ship this as is

**Pull request merged**